services/horizon/internal/integration/invokehostfunction_test.go

```diff
@@ -346,6 +377,22 @@ func assembleInstallContractCodeOp(t *testing.T, sourceAccount string, wasmFileN
 	}
 }
 
+func verifyEmptySorobanMeta(t *testing.T, clientTx horizon.Transaction) {
```

Comment:
Is this no longer used anywhere?

Reply:
good call out, I meant to still use this when `DisabledSoroban=true` in the test cases, I'll push that up to your branch directly in a bit, as i'm stepping through that per our huddle.